app/helpers/application_helper.rb

```diff
@@ -170,9 +170,16 @@ def get_countries_search_config
     }.to_json
   end
 
+  def get_countries_hero_search_config
+    {
+      id: 'country',
+      placeholder: 'Search for Country or Territory'
```

Comment:
Do you know if this is a standard input element or does it use some custom logic which requires it to be set like this? If it is a standard input I think you can just set the `placeholder` attribute directly on the Vue component

Reply:
Removed, and hard coded with translation